user/views.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from rest_framework import generics
+from rest_framework.authtoken.views import ObtainAuthToken
+from rest_framework.settings import api_settings
+from rest_framework.authentication import TokenAuthentication
+from rest_framework.permissions import IsAuthenticated
+
+
+from user.serializers import UserSerializer
+
+
+class CreateUserView(generics.CreateAPIView):
+    serializer_class = UserSerializer
```

Comment:
The `CreateUserView` correctly uses the `UserSerializer` to handle user creation.